src/components/Filter.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classnames from 'classnames';
+import { Status } from '../types/Status';
+
+interface FilterProps {
+  currentFilterStatus: Status;
+  onFilter: (status: Status) => void;
+}
+
+export const Filter: React.FC<FilterProps> = ({
+  currentFilterStatus,
+  onFilter,
+}) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      {Object.values(Status).map(field => (
+        <a
+          key={field}
+          onClick={() => onFilter(field)}
+          href="#/"
+          className={classnames('filter__link', {
+            selected: field === currentFilterStatus,
+          })}
+          data-cy={`FilterLink${field}`}
+        >
+          {field.toUpperCase()}
```

Comment:
Why it's uppercased? It should be capitalized, and you can do it with `css`(https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/63243136/015cf839-e477-4474-b3de-a220dbcf5f7e)